cmd/nerdctl/ipfs_linux_test.go

```diff
@@ -134,13 +133,11 @@ func pushImageToIPFS(t *testing.T, base *testutil.Base, name string, opts ...str
 
 func cidOf(t *testing.T, lines []string) string {
 	assert.Equal(t, len(lines) >= 2, true)
-	c, err := cid.Decode(lines[len(lines)-2])
-	assert.NilError(t, err)
-	return "ipfs://" + c.String()
+	return "ipfs://" + lines[len(lines)-2]
 }
 
 func requiresIPFS(t *testing.T) {
```

Comment:
maybe put in `testutil`, similar to other `RequireXXXX`.

https://github.com/containerd/nerdctl/blob/e2d618f283372f19c4fc52b809d6f6e1944ef8d3/pkg/testutil/testutil.go